lib/jsdom/living/nodes/HTMLElement-impl.js

```diff
@@ -86,6 +106,12 @@ class HTMLElementImpl extends ElementImpl {
   }
 }
 
+function callHandler(event, name) {
```

Comment:
Maybe name this something like `callEventBehaviorHook` or something? Handler is too easy to confuse with actual event handlers.